### PR TITLE
Issue/7241 update site button

### PR DESCRIPTION
**Pull request by theck13**
### Fix
Update layout of email login screen to make "Log in by entering your site address." and "Log in with your username." buttons more prominent as described in https://github.com/wordpress-mobile/WordPress-Android/issues/7241.  See the screenshots below for illustration.

update site button

### Test
***Default***

0. Clear app data.
1. Tap ***Log In*** button.
2. Notice site address login button is shown.

***Stats***

0. Log in with self-hosted account without Jetpack connected.
1. Go to ***Sites*** tab.
2. Tap ***Stats*** item.
3. Notice username login button is shown.

***Link***

0. Clear app data.
1. Tap deep link to WordPress post.
2. Notice neither site address nor username login button is shown.

### CODE REVIEW

**Comment by folletto** [comment]
Design wise 👍

File: libs/login/WordPressLoginFlow/src/main/res/layout/login_email_screen.xml [hunk 1]
```diff
@@ -35,49 +35,82 @@
...
+            app:srcCompat="@drawable/ic_google_60dp" >
+        </ImageView>
 
         <TextView
             android:gravity="center_vertical"
             android:layout_height="match_parent"
             android:layout_width="wrap_content"
-            android:text="@string/login_google_button_prefix"
-            style="@style/LoginTheme.TextLabel" >
+            android:text="@string/login_google_button_suffix"
```
Comment (hypest) [inline libs/login/WordPressLoginFlow/src/main/res/layout/login_email_screen.xml]: Oh, I like that we simplified the text+icon+text into just icon+text!

Can we perhaps remove the `login_google_button_prefix` string resource now?

Comment (theck13) [reply libs/login/WordPressLoginFlow/src/main/res/layout/login_email_screen.xml]: Yes, we can.  I removed the string resource in aa1e0327d68e19796351ec425936e0cfc29b21d5.

**Comment by hypest** [comment]
LGTM!

**Pull request merged**